controls/radcallout/key-properties.md

```diff
@@ -12,11 +12,23 @@ position: 3
 
 This article will list the key properties provided by the RadCallout control.
 
-* __ArrowAnchorPoint__: A property of type __Point__ that gets or sets the anchor point of the arrow geometry. 
+### __Geometry customization__
 
-* __ArrowBasePoint1__: A property of type __Point__ that gets or sets the first base point of the arrow geometry.
+The RadCallout provides 3 properties for customizing the arrow of its geometry - ArrowBasePoint1 , ArrowBasePoint2 and ArrowAnchorPoint.
 
-* __ArrowBasePoint2__: A property of type __Point__ that gets or sets the second base point of the arrow geometry.
+* __ArrowAnchorPoint__: A property of type __Point__ that gets or sets the __anchor point__ of the arrow geometry. 
+* __ArrowBasePoint1__: A property of type __Point__ that gets or sets the __first base point__ of the arrow geometry.
+* __ArrowBasePoint2__: A property of type __Point__ that gets or sets the __second base point__ of the arrow geometry.
+
+They are of type Point and represent relative coordinates (between 0 and 1) where for example 
+X = 0 and Y = 0 means the topleft point of the callout body and X= 1, Y = 1 means the 'bottom-right'
+point. In the figures you can see the default values of the properties [0.25, 0.5], [0.75, 0.5] and [0.5, 1.25].
+
+> Important to note here is that the widht and height of the callout applies only to the body of the geometry 
```

Comment:
"applies" should be "apply"

"..making big arrow exceeding the shape size". I am not sure for this one, but it sounds like it should be "..making __the__ big arrow.."